saving/game_save_manager.py

```diff
@@ -0,0 +1,184 @@
+# The prefix attached to save files created by the autosave feature.
+import json
+import os
+from datetime import datetime
+from itertools import chain
+
+import pyxel
+
+from board import Board
+from calculator import clamp
+from catalogue import get_blessing, get_project, get_unit_plan, get_improvement
+from game_management.game_controller import GameController
+from game_management.game_state import GameState
+from models import Biome, Heathen, UnitPlan, AttackPlaystyle, AIPlaystyle, Unit, ExpansionPlaystyle, Faction
+from saving.save_encoder import SaveEncoder, ObjectConverter
+
+AUTOSAVE_PREFIX = "auto"
+# The directory where save files are created and loaded from.
+SAVES_DIR = "saves"
+
+
+class GameSaveManager:
```

Comment:
Same here with the no-class approach, I think it would work here as well.

Reply:
Sounds good, will do